tests/testthat/test-plot.R

```diff
@@ -0,0 +1,15 @@
+### Test for plot() occumbFit --------------------------------------------------
+test_that("plot() works for occumbFit as expected", {
```

Comment:
Since the test does not testing the expected behavior, it would be better to title it like "plot() has known output for occumbFit".